addons/account/models/account_move.py

```diff
@@ -614,6 +614,14 @@ def _sequence_year_range_monthly_regex(self):
         tracking=True,
         help="It indicates that the invoice/payment has been sent or the PDF has been generated.",
     )
+    is_move_sent_string = fields.Selection(
```

Comment:
In python it's kind of a bad practice to name a field with his type, why ? Because the type can change at any time during the execution. So maybe we could name the field move_sent_values